modules/dbparser/pdb-file.c

```diff
@@ -77,3 +79,88 @@ pdb_file_detect_version(const gchar *pdbfile, GError **error)
     }
   return result;
 }
+
+static const gchar *
+_get_xsddir_in_build(void)
+{
+#ifdef SYSLOG_NG_ENABLE_DEBUG
+  const gchar *srcdir;
+  static gchar path[256];
```

Comment:
A small note here:
Maybe we could check if `PATH_MAX` is available or not,  if not then we could try to use pathconf (a separate `build_path(base_dir, relative_path)` function could be also added).

(IIRC I have a patch somewhere for determining path max ... maybe in PE).

This is not a blocker issue, it can be merged as-is (and later I'll send a patch).